HospitalToday/Repository/PersonRepository.cs

```diff
@@ -1,48 +1,59 @@
-﻿using System;
+﻿using HospitalToday.Domain;
+using HospitalToday.Domain.Person;
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 
-namespace HospitalToday.Domain
+namespace HospitalToday.Repository
 {
     class PersonRepository : IRepository<Person>
     {
-        List<Person> Persons;
+        private static IRepository<Person> personRep;
+        private List<Person> persons;
 
-        public PersonRepository()
+        private PersonRepository()
         {
-            Persons = new List<Person>();
+            persons = new List<Person>();
+        }
+
+        public static IRepository<Person> GetRepository()
+        {
+            if (personRep == null)
+                personRep = new PersonRepository();
+
+            return personRep;
         }
 
         public void Create(Person item)
         {
-            Persons.Add(item);
+            persons.Add(item);
         }
 
         public void Delete(int id)
         {
-            Persons.Remove(Persons.Where(p => p.Id == id).First());
+            persons.Remove(persons.FirstOrDefault(p => p.Id == id));
```

Comment:
Please, implement check as we have in other Rep.

Reply:
Fixed. I did it.